libr/anal/p/anal_evm_cs.c

```diff
@@ -64,19 +64,41 @@ static int analop(RAnal *anal, RAnalOp *op, ut64 addr, const ut8 *buf, int len,
 		goto beach;
 	}
 	if (mask & R_ANAL_OP_MASK_DISASM) {
-		char *str = r_str_newf ("%s%s%s", insn->mnemonic, insn->op_str[0]? " ": "", insn->op_str);
+		char *str = r_str_newf ("%s%s%s", insn->mnemonic, insn->op_str[0]? " 0x": "", insn->op_str);
```

Comment:
uhm, i think this change should be done in the disassembler itself, but i guess this would be harder to change than doing it in here. in op_str always a number? coulnt be a memory reference or a register?

Reply:
It is only constants and only for the push instructions. I found another problem with capstone: https://github.com/capstone-engine/capstone/pull/1231. I updated my code and tests to handle it.